ScaleformUI_Lua/src/Hud/Marker.lua

```diff
@@ -25,10 +25,6 @@ function Marker.New(type, position, scale, distance, color, placeOnGround, bobUp
 end
 
 function Marker:Draw()
-    -- [Position.Z != _height] means that we make the check only if we change position
```

Comment:
i'd leave this 3 lines to explain people reading my source code. don't delete them